lazyflow/operators/ioOperators/opExportMultipageTiff.py

```diff
@@ -17,246 +17,265 @@
-
-        parallel_requests = self.DEFAULT_BATCH_SIZE
-        
-        # If ram usage info is available, make a better guess about how many requests we can launch in parallel
-        ram_usage_per_requested_pixel = self.Input.meta.ram_usage_per_requested_pixel
-        if ram_usage_per_requested_pixel is not None:
-            pixels_per_slice = numpy.prod(shape_yx)
-            if 'c' in tagged_shape:
-                pixels_per_slice //= tagged_shape['c']
-            
-            ram_usage_per_slice = pixels_per_slice * ram_usage_per_requested_pixel
-
-            # Fudge factor: Reduce RAM usage by a bit
-            available_ram = psutil.virtual_memory().available
-            available_ram *= 0.5
-
-            parallel_requests = int(available_ram // ram_usage_per_slice)
-        
-        # Start with a batch of images
-        reqs = collections.deque()
-        for next_request_index in range( min(parallel_requests, num_pages) ):
-            reqs.append( next(iter_slice_requests) )
-        
-        self.progressSignal(0)
-        pages_written = 0
-        while reqs:
-            self.progressSignal( 100*next_request_index // num_pages )
-            req = reqs.popleft()
-            slice_data = req.wait()
-            slice_data = vigra.taggedView(slice_data, self._export_axes)
-            next_request_index += 1
-            
-            # Add a new request to the batch
-            if next_request_index < num_pages:
-                reqs.append( next(iter_slice_requests) )
-            
-            if pages_written == 0:
-                xml_description = OpExportMultipageTiff.generate_ome_xml_description(
-                                     self._opReorderAxes.Output.meta.getAxisKeys(),
-                                     self._opReorderAxes.Output.meta.shape,
-                                     self._opReorderAxes.Output.meta.dtype,
-                                     os.path.split(image_path)[1] )
-                # Write the first slice with tifffile, which allows us to write the tags.
-                with tifffile.TiffWriter( image_path, software='ilastik', byteorder='<' ) as writer:
-                    writer.save( slice_data.withAxes('yx'), description=xml_description, planarconfig='planar')
-            else:
-                # Append a slice to the multipage tiff file
-                vigra.impex.writeImage( slice_data.withAxes('yx'), image_path, dtype='', compression='NONE', mode='a' )
-            pages_written += 1
+    def execute(self, slot, subindex, roi, result):
+        pass
 
-        self.progressSignal(100)
+    def propagateDirty(self, slot, subindex, roi):
+        pass
 
-    # No output slots...
-    def execute(self, slot, subindex, roi, result): pass 
-    def propagateDirty(self, slot, subindex, roi): pass
+    def run_export(self) -> None:
+        """Export an image from Input to Filepath."""
+        path = pathlib.Path(self.Filepath.value)
```

Comment:
I've used `pathlib` instead of `os` for something else, and just didn't want to `import os`, and also it's API is nicer and less error-prone. In this case it doesn't really matter, but I think it's better to use `pathlib` in new code anyway for consistency.